crates/rome_js_analyze/src/analyzers/correctness/no_unreachable_super.rs

```diff
@@ -121,25 +124,20 @@ impl Rule for NoUnreachableSuper {
         // As soon as we detect the use of this without calling first `super`
 
         // stack of block contexts to process
-        let mut block_context_stack = Vec::new();
-        let mut visited_block_contexts = FxHashSet::default();
-        block_context_stack.push(BlockContext {
-            block_index: 0,
+        let mut block_context_stack = vec![BlockContext {
+            block_id: FIRST_BLOCK_ID,
             super_call: None,
-        });
-        // Set of couples (block_index, was_visited_with_super_call)
-        visited_block_contexts.insert((0u32, false));
+        }];
+        let mut visited_block_contexts = FxHashSet::default();
+        // Set of couples (block_id, was_visited_with_super_call)
+        visited_block_contexts.insert((FIRST_BLOCK_ID, false));
```

Comment:
You could initialize `visited_block_contexts` directly with `(FIRST_BLOCK_ID, false)`,  right?

Reply:
With `from_iter`? Not sure it is better?